frontend/packages/topology/src/actions/provider.ts

```diff
@@ -17,3 +22,39 @@ export const useTopologyWorloadActionProvider = (element: GraphElement) => {
     return [actions, true, undefined];
   }, [actions, element]);
 };
+
+export const useTopologyContextActionProvider = (element: GraphElement) => {
```

Comment:
I would suggest moving these action providers to dev-console package.

Reply:
Moved providers to dev console package.